### DISCUSSION TITLE
SNES Direct Console Plug Wrong Button Mapping Switch Pro/Switch SNES

### DISCUSSION THREAD

**Discussion by theskipster00** [darthcloud/BlueRetro#491] [Q&A]
Heyo! Having an issue with using BlueRetro directly with an SNES plug. I successfully did the same thing with an N64 plug, but can't get it to work with SNES. Using a normal wired controller does result in the correct inputs, so I don't think it's on the console side. I tried to fix it in the button mapping config as a work around, but can't find anything that activates the B button. I have tried with both a Switch Pro and Switch SNES controller, which result in the same incorrect button mapping. I have quadruple checked that I mapped the correct wires to the correct  pins on the SNES plug. I have connected 5V to 5V on the ESP32 board, GND to GND, P1_CLK to IO5, LATCH to IO32, P1_D0 to IO19, and IO18 to 3.3V as the instructions stated. I tried flashing both the SNES bin and universal bin. I double checked I was flashing the correct bin files with the correct offsets. Any idea what could be wrong?

**Comment by darthcloud** [comment]
Can you post aome pictures of how you wired things?
